app/src/styles/StyleCardView.tsx

```diff
@@ -0,0 +1,82 @@
+import { styled } from '@mui/material/styles';
+import { Box, Typography } from '@mui/material';
+import ButtonLink from '@/components/ButtonLink';
+
+// Color Scheme:
+// #ECF2F8: Light blue
+// #96b6d4: Mild blue
+// #477096: Dark blue
+// #4F4846: Black
+// #FDF8F5: White
+
+// Define custom styles
+const StyledReportDetails = styled(Box)({
+    '& p': {
+        margin: 0,
+        padding: '5px 0',
+        color: '#4F4846',
+    },
+    '& strong': {
+        color: '#4F4846',
+        textAlign: 'center',
+    },
+    borderRadius: '2px',
+    background: '#FDF8F5',
+    boxShadow: '0px 0px 5px rgba(0, 0, 0, 0.3)',
+});
+
+const Box1 = styled(Box)({
```

Comment:
Non-blocking: "Box1" and "Box2" are not descriptive names, devs have to look very carefully at the code to understand the difference :) Could you please give them more descriptive variable names?